app/views/users/logix/favourites.html.erb

```diff
@@ -8,8 +8,13 @@
   <div class ="row">
     <h2 class="card-title">Favourite Projects</h2>
   </div>
-
-  <div class="row ">
+  <% ratedproject = @user.rated_projects %>
+    <% if ratedproject.count == 0 %>
```

Comment:
no need to store this in a temp variable, just use if @user.projects.blank?

Reply:
But @tachyons here it is asking for rated_projects (Favourites) so shouldn't be @user.rated_projects.blank ?

Reply:
Yes, I was just suggesting the syntax